.github/workflows/release.yaml

```diff
@@ -172,18 +197,18 @@ jobs:
           echo "* Desired release branch: ${{ steps.get-version.outputs.release-branch-name }}" >> $GITHUB_STEP_SUMMARY
           echo "* Release date: $(date)" >> $GITHUB_STEP_SUMMARY
       - name: Release radius-project/radius version ${{ steps.get-version.outputs.release-version }}
+        if: success() && env.STOPPED == 'false'
         run: |
           ./radius/.github/scripts/release-create-tag-and-branch.sh radius ${{ steps.get-version.outputs.release-version }} ${{ steps.get-version.outputs.release-branch-name }}
-        if: success()
       - name: Release radius-project/bicep version ${{ steps.get-version.outputs.release-version }}
+        if: success() && env.STOPPED == 'false'
         run: |
           ./radius/.github/scripts/release-create-tag-and-branch.sh bicep ${{ steps.get-version.outputs.release-version }} ${{ steps.get-version.outputs.release-branch-name }}
-        if: success()
       - name: Release radius-project/deployment-engine version ${{ steps.get-version.outputs.release-version }}
+        if: success() && env.STOPPED == 'false'
         run: |
           ./radius/.github/scripts/release-create-tag-and-branch.sh deployment-engine ${{ steps.get-version.outputs.release-version }} ${{ steps.get-version.outputs.release-branch-name }}
-        if: success()
       - name: Release radius-project/recipes version ${{ steps.get-version.outputs.release-version }}
+        if: success() && env.STOPPED == 'false'
```

Comment:
Question: The only way we can test this is in the upcoming release?is that right?

Reply:
correct, unfortunately

Reply:
the good news is that if it fails during the release, we can just do this stuff manually (as we do today)